custom_components/alexa_media/__init__.py

```
@@ -824,7 +824,59 @@ async def login_success(event=None) -> None:
+                                ):
+                                    _LOGGER.debug(
+                                        "[BOOT] OAuth token refresh succeeded"
+                                    )
+                                    login.status["login_successful"] = True
+                                    login.customer_id = auth.get("customerId")
+                                    login.stats["login_timestamp"] = datetime.now()
+                                    login.stats["api_calls"] = 0
+                                    await login.check_domain()
+                                    await login.finalize_login()
+                                    cookie_login_ok = True
+            except Exception as ex:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)